### ISSUE TITLE
Very slow and very low accuracy

### ISSUE THREAD

**Issue by demetrii** [googledatalab/notebooks#152]
Hi,

After multiple test runs, my output is very poor. I was wondering if anyone could explain to me why this is and/or how to tackle this problem. As input I am using "europarl-v7.nl-en.nl" which is a Dutch parliamentary text of 327MB. In my eyes, this is large enough to get at least moderate results. Please see below the results.

WARNING:tensorflow:From LSTM_copy.py:298: get_or_create_global_step (from tensorflow.contrib.framework.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Please switch to tf.train.get_or_create_global_step
WARNING:tensorflow:From LSTM_copy.py:415: Supervisor.__init__ (from tensorflow.python.training.supervisor) is deprecated and will be removed in a future version.
Instructions for updating:
Please switch to tf.train.MonitoredTrainingSession
2018-02-19 16:41:26.997355: I C:\tf_jenkins\workspace\rel-win\M\windows\PY\36\tensorflow\core\platform\cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2

Epoch: 1 Learning rate: 1.000
Epoch: 1 Train Perplexity: 1.002
. --- total: 1031, correct: 0, accuracy: 0.000,
, --- total: 632, correct: 0, accuracy: 0.000,
? --- total: 55, correct: 0, accuracy: 0.000,
! --- total: 9, correct: 0, accuracy: 0.000,
: --- total: 26, correct: 0, accuracy: 0.000,
n --- total: 9883247, correct: 9883247, accuracy: 1.000,
Epoch: 1 Valid Perplexity: 1.002
Epoch: 2 Learning rate: 1.000
Epoch: 2 Train Perplexity: 1.002
. --- total: 1032, correct: 0, accuracy: 0.000,
, --- total: 636, correct: 0, accuracy: 0.000,
? --- total: 55, correct: 0, accuracy: 0.000,
! --- total: 9, correct: 0, accuracy: 0.000,
: --- total: 26, correct: 0, accuracy: 0.000,
n --- total: 9883242, correct: 9883242, accuracy: 1.000,
Epoch: 2 Valid Perplexity: 1.002
Epoch: 3 Learning rate: 1.000
Epoch: 3 Train Perplexity: 1.002
. --- total: 1032, correct: 0, accuracy: 0.000,
, --- total: 637, correct: 0, accuracy: 0.000,
? --- total: 55, correct: 0, accuracy: 0.000,
! --- total: 9, correct: 0, accuracy: 0.000,
: --- total: 26, correct: 0, accuracy: 0.000,
n --- total: 9883241, correct: 9883241, accuracy: 1.000,
Epoch: 3 Valid Perplexity: 1.002
Epoch: 4 Learning rate: 1.000
Epoch: 4 Train Perplexity: 1.002
. --- total: 1032, correct: 0, accuracy: 0.000,
, --- total: 636, correct: 0, accuracy: 0.000,
? --- total: 55, correct: 0, accuracy: 0.000,
! --- total: 9, correct: 0, accuracy: 0.000,
: --- total: 26, correct: 0, accuracy: 0.000,
n --- total: 9883242, correct: 9883242, accuracy: 1.000,
Epoch: 4 Valid Perplexity: 1.002
Epoch: 5 Learning rate: 1.000
Epoch: 5 Train Perplexity: 1.002
. --- total: 1032, correct: 0, accuracy: 0.000,
, --- total: 636, correct: 0, accuracy: 0.000,
? --- total: 55, correct: 0, accuracy: 0.000,
! --- total: 9, correct: 0, accuracy: 0.000,
: --- total: 26, correct: 0, accuracy: 0.000,
n --- total: 9883242, correct: 9883242, accuracy: 1.000,
Epoch: 5 Valid Perplexity: 1.002
Epoch: 6 Learning rate: 1.000
Epoch: 6 Train Perplexity: 1.002
. --- total: 1029, correct: 0, accuracy: 0.000,
, --- total: 637, correct: 0, accuracy: 0.000,
? --- total: 55, correct: 0, accuracy: 0.000,
! --- total: 9, correct: 0, accuracy: 0.000,
: --- total: 26, correct: 0, accuracy: 0.000,
n --- total: 9883244, correct: 9883244, accuracy: 1.000,
Epoch: 6 Valid Perplexity: 1.002
Epoch: 7 Learning rate: 1.000
Epoch: 7 Train Perplexity: 1.001
. --- total: 1031, correct: 0, accuracy: 0.000,
, --- total: 634, correct: 0, accuracy: 0.000,
? --- total: 55, correct: 0, accuracy: 0.000,
! --- total: 9, correct: 0, accuracy: 0.000,
: --- total: 26, correct: 0, accuracy: 0.000,
n --- total: 9883245, correct: 9883245, accuracy: 1.000,
Epoch: 7 Valid Perplexity: 1.002
Epoch: 8 Learning rate: 1.000
Epoch: 8 Train Perplexity: 1.001
. --- total: 1032, correct: 0, accuracy: 0.000,
, --- total: 637, correct: 0, accuracy: 0.000,
? --- total: 55, correct: 0, accuracy: 0.000,
! --- total: 9, correct: 0, accuracy: 0.000,
: --- total: 26, correct: 0, accuracy: 0.000,
n --- total: 9883241, correct: 9883241, accuracy: 1.000,
Epoch: 8 Valid Perplexity: 1.001
Epoch: 9 Learning rate: 1.000
Epoch: 9 Train Perplexity: 1.001
. --- total: 1033, correct: 1, accuracy: 0.001,
, --- total: 637, correct: 0, accuracy: 0.000,
? --- total: 55, correct: 0, accuracy: 0.000,
! --- total: 9, correct: 0, accuracy: 0.000,
: --- total: 26, correct: 0, accuracy: 0.000,
n --- total: 9883240, correct: 9883240, accuracy: 1.000,
Epoch: 9 Valid Perplexity: 1.001
Epoch: 10 Learning rate: 1.000
Epoch: 10 Train Perplexity: 1.001
. --- total: 1031, correct: 3, accuracy: 0.003,
, --- total: 636, correct: 0, accuracy: 0.000,
? --- total: 55, correct: 0, accuracy: 0.000,
! --- total: 9, correct: 0, accuracy: 0.000,
: --- total: 26, correct: 0, accuracy: 0.000,
n --- total: 9883243, correct: 9883243, accuracy: 1.000,
Epoch: 10 Valid Perplexity: 1.001
